src/Components/Bodies/CodeBodyVersionized.tsx

```diff
@@ -267,12 +256,19 @@ class CodeBodyVersionized extends Component<Props, ICode> {
                 });
             }, {});
 
-        let attributesHtml = Object.keys(codeAttributes).map((attribute, i) => {
+        let attributesHtml = Object.keys(codeAttributes).map((attribute) => {
             let attributeValue = codeAttributes[attribute];
             if (typeof attributeValue === 'object') {
-                return this.objectTypeCodeAttributes(translateJson, attribute, i, attributeValue)
+                return <div key={attribute}>
+                    <h5>{translateJson["LBL_" + attribute.toUpperCase()]}</h5>
+                    <ul>
+                        {attributeValue.map((val, j) => (
+                            this.htmlListItem(attribute, val, j)
```

Comment:
Same here: See review for #15 : Try to put all JAX / HTML structure in one place for each file so it is more readable.